core/services/relay/relay.go

```diff
@@ -28,6 +31,15 @@ var SupportedNetworks = map[string]struct{}{
 	NetworkDummy: {},
 }
 
+// From returns a NetworkType from a string.
+func From(s string) (NetworkType, error) {
+	s = strings.ToLower(s)
```

Comment:
I don't think it is a good idea to do fuzzy matching here

Reply:
Not necessarily. Even if we only used lower case values, I would still object to this kind of fuzzy match, because it lets bad data in to the system. Unless there is a strict need to do this conversion, it only introduces risk.